### DISCUSSION TITLE
Install 0.51.0-BETA9 Update error

### DISCUSSION THREAD

**Discussion by GowriShanakar** [rejetto/hfs#448] [General]
Cant update in windows 11, screen
![HFS 3 UPDATE ERROR](https://github.com/rejetto/hfs/assets/90929359/c7de6bc9-8708-4398-b64f-8e30df482d24)
 shot attached

**Comment by rejetto** [comment]
hey, that is saying to check your server.
Did you check your server? What did you find?

anyway, if the automatic update fails, you can still update by unzipping and overwriting the program.
